Mirror/Editor/NetworkAnimatorEditor.cs

```diff
@@ -83,11 +83,13 @@ void DrawControls()
             if (Application.isPlaying)
             {
                 EditorGUILayout.Separator();
-                if (m_AnimSync.param0 != "") EditorGUILayout.LabelField("Param 0", m_AnimSync.param0);
-                if (m_AnimSync.param1 != "") EditorGUILayout.LabelField("Param 1", m_AnimSync.param1);
-                if (m_AnimSync.param2 != "") EditorGUILayout.LabelField("Param 2", m_AnimSync.param2);
-                if (m_AnimSync.param3 != "") EditorGUILayout.LabelField("Param 3", m_AnimSync.param3);
-                if (m_AnimSync.param4 != "") EditorGUILayout.LabelField("Param 4", m_AnimSync.param4);
+                for (int index = 0; index < 5; index++)
```

Comment:
why not param.Length instead of 5?

Reply:
Cause param5 wasn't processed there earlier